spec/features/admin/unit_price_spec.rb

```diff
@@ -0,0 +1,50 @@
+# frozen_string_literal: true
+
+require 'spec_helper'
+
+feature '
+    As an admin
+    I want to check the unit price of my products/variants
+' do
+  include AuthenticationHelper
+  include WebHelper
+  
+  let!(:stock_location) { create(:stock_location, backorderable_default: false) }
+  
+  before do
+    allow(OpenFoodNetwork::FeatureToggle).to receive(:enabled?).with(:unit_price, anything) { true }
+  end
+  
+  describe "product", js: true do
+    scenario "creating a new product", :debug do
+      login_as_admin_and_visit spree.admin_products_path
+      click_link 'New Product'
+      select "Weight (kg)", from: 'product_variant_unit_with_scale'
+      fill_in 'product_unit_value_with_description', with: '1'
```

Comment:
It's generally better to use the label instead of the name of the input because that's how the user finds it on the page.
```suggestion
      fill_in "Value", with: '1'
```

Reply:
No need to pay attention with i18n at this point?

Reply:
Actually, on this particular field, it has one label for 3 <inputs />, I guess it's not reliable.

Reply:
Okay, it would probably good if the label and input were linked. It's about out of scope here but it's always good to improve the code when you encounter something like this. Just keep in a separate commit so that it's clear that you are fixing a different problem.

Reply:
Yes, you're right. I was afraid to change something like this, but anyway: I'll do it!

Reply:
Done! ;)